docker-hub/api/dvp.yaml

```diff
@@ -0,0 +1,564 @@
+openapi: 3.0.0
+info:
+  title: DVP Data API
+  version: 1.0.0
+  x-logo:
+    url: https://docs.docker.com/images/logo-docker-main.png
+    href: /reference
+  description: |
+    The Docker DVP Data API allows Docker Verified Publishers to view image pull analytics data for their namespaces
```

Comment:
Should we add a link to the DVP programme? (https://docs.docker.com/docker-hub/publish/)

Reply:
Sure, no harm in that.